### DISCUSSION TITLE
Multimodal Image Registration

### DISCUSSION THREAD

**Discussion by kenrickschulze** [Project-MONAI/MONAI#7518] [Q&A] [answered]
Dear community,

I'd like to dive into deep learning for registering images.
I have a multimodal dataset with varying pixel spacing and dimensions for pairs of still and moving images, and across the entire dataset as nrrd files.

My understanding of the DL Reg workflow is that you first have to bring all the images into a common shape.
Can anyone advise me on how to go about doing this in Monai? 

I am mainly confused as to whether to apply only "Resized" or also "Spacing". And if I should use both of them, in what order should they be applied?

I would really appreciate any help on how to do this and would be even more grateful if you could explain why I should do it a certain way.

**Comment by KumoLiu** [comment, marked as answer]
Hi @kenrickschulze, thanks for your interest here.

We currently offer several tutorials that delve into utilizing MONAI for registration tasks. For further guidance, feel free to explore the following resources:
- [3D Registration Tutorial](https://github.com/Project-MONAI/tutorials/tree/main/3d_registration)
- [2D Registration Tutorial](https://github.com/Project-MONAI/tutorials/tree/main/2d_registration)

As for your query regarding whether to apply "Resized" and/or "Spacing", it is typically advised to commence by harmonizing the spacing of your data. The order really depends on your specific use case, but unifying the spacing as a first step is a common practice.

Hope it helps, thanks.

**Reply by kenrickschulze** [reply]
Thank you :) I checked the tutorials but none mentioned the spacing, as far as i saw it.
Everything else was super clear to me, the tutorials are great !

It helps a lot!
